### PR TITLE
implements subtraction function

### PR DESCRIPTION
**Pull request by AqeelWWU**

### CODE REVIEW

File: collaboration.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
 def add(a, b):
     return a+b
+
+def sub(a, b):
+    return a-b   
+    
 print("addition result",add(4,5))
+print("subtraction result",sub(4,5))
```
Comment (letBeDev26) [inline collaboration.py]: @AqeelWWU Please add newline here

File: collaboration.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
 def add(a, b):
     return a+b
+
+def sub(a, b):
```
Comment (letBeDev26) [inline collaboration.py]: Please add proper variable names

Comment (letBeDev26) [inline collaboration.py]: Add function description

Comment (letBeDev26) [inline collaboration.py]:
```suggestion
def sub(first_var, second_var):
```

**Review by letBeDev26 — CHANGES_REQUESTED** [review]
Please make your chages

File: collaboration.py (hunk 2 above)
Comment (AqeelWWU) [reply collaboration.py]: I have reviwed the code.

**Pull request merged**